### PR TITLE
cardano node bump 1.15.0

### PR DESCRIPTION
**Pull request by paweljakubas**
# Issue Number

# Overview

- [x] I have changed yaml
- [x] I have updated matrix
- [x] I have updated our code to make it compiling
- [ ] All test passes

# Comments

### CODE REVIEW

**Comment by paweljakubas** [comment]
I have still when `stack build` : 
```
cardano-wallet-byron  > /home/pawel/Work/IOHK/cardano-wallet/lib/byron/src/Cardano/Wallet/Byron/Compatibility.hs:329:45: error:
cardano-wallet-byron  >     • No instance for (O.HasFullHeader ByronBlock)
cardano-wallet-byron  >         arising from a use of 'blockPrevHash'
cardano-wallet-byron  >     • In the second argument of '($)', namely 'blockPrevHash byronBlk'
cardano-wallet-byron  >       In the 'parentHeaderHash' field of a record
cardano-wallet-byron  >       In the 'header' field of a record
cardano-wallet-byron  >     |
cardano-wallet-byron  > 329 |                 fromChainHash genesisHash $ blockPrevHash byronBlk
```

**Comment by rvl** [comment]
Seems to build ok with your changes @paweljakubas.

bors try

File: lib/shelley/src/Cardano/Wallet/Shelley/Launch.hs [hunk 1]
```diff
@@ -899,76 +899,76 @@ takeFaucet dir = takeMVar faucets >>= \case
...
-      , unlines
-          [ "type: Node operational certificate issue counter"
-          , "title: Next certificate issue number: 0"
-          , "cbor-hex:"
-          , " 00"
+      , Aeson.object
+          [ "type" .= Aeson.String "Node operational certificate issue counter"
+          , "description" .= Aeson.String "Next certificate issue number: 0"
+          , "cborHex" .= Aeson.String
+              "00"
```
Comment (KtorZ) [inline lib/shelley/src/Cardano/Wallet/Shelley/Launch.hs]: I'll have a look at that one :point_up:

**Comment by paweljakubas** [comment]
@rvl indeed, `stack build` works. Also
```
stack test cardano-wallet-byron:unit
stack test cardano-wallet-shelley:unit
stack test cardano-wallet-byron:integration
```
works.
Problem is with `stack test cardano-wallet-shelley:integration`

**Review by KtorZ — APPROVED** [review]
:heavy_check_mark:

**Comment by KtorZ** [comment]
bors retry

**Comment by KtorZ** [comment]
Failure seems benign (as only occurring for one of the environment).

Reporting and merging manually.

**Pull request merged**